### PR TITLE
Attempting to turn back some tests on

### PR DESCRIPTION
**Pull request by AshantiMutinta**
## testing/benchmarking notes

( if any manual testing or benchmarking was/should be done, add notes and/or screenshots here )

## followups

( any new tickets/concerns that were discovered or created during this work but aren't in scope for review here )

## changelog

Please check one of the following, relating to the [CHANGELOG-UNRELEASED.md](https://github.com/holochain/holochain-rust/blob/develop/CHANGELOG-UNRELEASED.md)

- [ ] this is a code change that effects some consumer (e.g. zome developers) of holochain core so it is added to the CHANGELOG-UNRELEASED.md (linked above), with the format `- summary of change [PR#1234](https://github.com/holochain/holochain-rust/pull/1234)`
- [x] this is not a code change, or doesn't effect anyone outside holochain core development

### CODE REVIEW

File: core/src/workflows/hold_link.rs [hunk 1]
```diff
@@ -127,37 +122,32 @@ pub mod tests {
...
-        // Address of the original DNA
-        let dna_address = dna.address();
 
-        let (_, context1) =
-            test_instance_with_spoofed_dna(hacked_dna, dna_address, "alice").unwrap();
-        let netname = Some("test_reject_invalid_link_on_remove_workflow");
+        let mut dna = test_dna();
+        dna.uuid = "test_reject_invalid_link_on_hold_workflow".to_string();
+        let netname = Some("test_reject_invalid_link_on_hold_workflow, the network");
+        let (_instance1, context1) = instance_by_name("jill", dna.clone(), netname);
```
Comment (maackle) [inline core/src/workflows/hold_link.rs]: Hmm is this really the same test? In the new version you're cloning the same DNA twice, but in the old it seems to use a hacked version that always validates, which is missing in the new version.

Comment (AshantiMutinta) [reply core/src/workflows/hold_link.rs]: This test doesn't run, I was trying to fix it but after further looking at it, it was not worth it atm

File: core/src/workflows/hold_link.rs [hunk 2]
```diff
@@ -128,7 +123,7 @@ pub mod tests {
     /// hold_link_workflow is then expected to fail in its validation step
     fn test_reject_invalid_link_on_hold_workflow() {
         // Hacked DNA that regards everything as valid
-        let hacked_dna = create_test_dna_with_wat("test_zome", Some(&test_wat_always_valid()));
+         let hacked_dna = create_test_dna_with_wat("test_zome", Some(&test_wat_always_valid()));
```
Comment (maackle) [inline core/src/workflows/hold_link.rs]: whitespace

File: core/src/workflows/hold_link.rs [hunk 3]
```diff
@@ -169,12 +165,13 @@ pub mod tests {
             entry: link_entry,
             header,
         };
-
+         println!("before hold link");
```
Comment (maackle) [inline core/src/workflows/hold_link.rs]: remove printlns?

Comment (AshantiMutinta) [reply core/src/workflows/hold_link.rs]: From the broken test still but will remove it

**Pull request merged**